R/content.R

```diff
@@ -1007,6 +1007,9 @@ content_render <- function(content, variant_key = NULL) {
     target_variant <- get_variant(content, variant_key)
   }
   render_task <- target_variant$render()
+  # Tasks returned from variant render endpoint look different from those
+  # returned by deploy.
+  render_task$task_id <- render_task$id
```

Comment:
I see this same logic in the Task `initialize` method: https://github.com/rstudio/connectapi/blob/main/R/deploy.R#L67-L69

Why doesn't that handle this for you already?

Reply:
Right, because of course `VariantTask` doesn't inherit from `Task` :/

I think the better fix would be to push this adapting into the `VariantTask` init method. Even better would be to look to fix the inheritance of all of these apparent (but not actual) Task subclasses, but that could be a followup--I'm not trying to derail a bugfix by tempting you into refactoring all the things.

Reply:
Yeah lol. But good point, I'll put it into `VariantTask` and rip out the other place that also does it.

I don't _think_ `ContentTask` needs it because I don't think there are any content APIs that return this form.